selfdrive/car/toyota/interface.py

```diff
@@ -120,7 +120,7 @@ def get_params(candidate, fingerprint):
       ret.steerKf = 0.00006
 
     elif candidate in [CAR.CAMRY, CAR.CAMRYH]:
```

Comment:
@daehahn , the non-hybrid camry uses the CHR dbc file, right? If so, this seems wrong

Reply:
Nice catch @rbiasini, I will confirm this with Slack community and get back to you later.

Reply:
Here it is after the change for **CAMRY**: https://community.comma.ai/cabana/?route=c0a607798e8eafa4%7C2018-09-19--23-10-17&max=13&url=https%3A%2F%2Fchffrprivate.blob.core.windows.net%2Fchffrprivate3%2Fv2%2Fc0a607798e8eafa4%2Fd0e200abbabeee7fda5165fee192b1bd_2018-09-19--23-10-17